tika-parsers/pom.xml

```diff
@@ -506,6 +506,52 @@
         <version>1.6.5</version>
         <scope>test</scope>
     </dependency>
+     <!--Deeplearning4j Dependencies for DL4JImageRecogniser-->
+    <dependency>
+      <groupId>org.deeplearning4j</groupId>
+      <artifactId>deeplearning4j-modelimport</artifactId>
+      <version>0.8.0</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.4</version>
+    </dependency>
+    <dependency>
+      <groupId>org.deeplearning4j</groupId>
+      <artifactId>deeplearning4j-core</artifactId>
+      <version>0.8.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.nd4j</groupId>
+      <artifactId>nd4j-native-platform</artifactId>
+      <version>0.8.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.deeplearning4j</groupId>
+      <artifactId>deeplearning4j-nn</artifactId>
+      <version>0.8.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.deeplearning4j</groupId>
+      <artifactId>dl4j-spark-ml</artifactId>
+      <version>0.4-rc3.8</version>
```

Comment:
Size of tika-app jar grew from 54MB to 263MB. I think lot of the dependencies that are added here are not necessary for *using* deeplearning4j (contrary to training model, optimizing for GPUs, cluster distributions etc ), and we can exclude them.
I am looking into this currently, more comments soon